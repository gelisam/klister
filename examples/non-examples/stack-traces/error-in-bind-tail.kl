#lang "prelude.kl"

(example
  (>>= (pure 'hello-go-boom)
    (lambda (x)
      (>>= (pure x)
      -- TODO: DYG: see #265
        (lambda (y) (error y))))))