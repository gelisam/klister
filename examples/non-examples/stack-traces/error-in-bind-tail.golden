pure #[error-in-bind-tail.kl:4.15-4.28]<hello-go-boom> >>= #<x> : ∀(α : *). (Macro α)
